packages/nextjs/mocks/factory/index.ts

```diff
@@ -0,0 +1,225 @@
+import faker from "faker";
```

Comment:
Thanks! I feel like this is the core to writing good tests ... You need an easy way to get generic data, and you need an easy way to get specific data. This provides both!
This can be used everywhere, e2e, integration, and unit tests, and even mock development.